src/VisualStudio/VisualStudioInteractiveComponents/VisualStudioInteractiveComponents.csproj

```diff
@@ -146,15 +146,20 @@
       </IncludeOutputGroupsInVSIXLocalOnly>
     </ProjectReference>
   </ItemGroup>
+  <!--
+    The VsSdk specifically excludes System.Collections.Immutable and System.Reflection.Metadata from
+    the output vsix, so we'll have to mark them with ForceIncludeInVSIX=true.  This is necessary so
+    that the InteractiveHost.exe process can load these assemblies (different from other extensions,
+    which are loaded into VS and get the versions of the dlls that VS has loaded).
+  -->
   <ItemGroup>
```

Comment:
Can you please include something in each individual reference?  It could be as simple as ```<!-- See above. -->```.